### ISSUE TITLE
Remove Cypress.wait() at the start of cypress tests.

### ISSUE THREAD

**Issue by gerhean** [reposense/RepoSense#1540]
**What feature(s) would you like to see in RepoSense?**
Remove Cypress.wait() at the start of cypress tests.

**Is the feature request related to a problem?**
None

**Additional context**

`Cypress.wait()` seems to be placed at the start of every cypress test, however they do not seem to serve any purpose but to slow down the tests.

Removing them might make the tests run faster.

**Comment by gerhean** [comment]
Will work on this soon once #1529 is merged if no objections.

**Comment by dcshzj** [comment]
I couldn't find the documentation for that particular function, but is it used to get Cypress to wait for the page to fully load before progressing with the test?

**Comment by Zhou-Jiahao-1998** [comment]
After some investigation, I found that `Cypress.wait()` is actually in the `support.js` file under `cypress` folder. The function simply waits for 1500 ms if it is in non-CI environment. I am not entirely sure of the reason for treating non-CI environment differently. However, this use of `wait` is an anti-pattern according to the [official documentation](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting). 

In short, the previous command will not resolve until a response is received (or timeout in a negative case). Therefore, there is no need to explicitly wait. Removing all the instances of `wait` did not cause any issue for me when running locally. My test time reduced from 135 seconds on average to 60 seconds on average now. This is a significant reduction. It should not affect the performance on the CI side as explained earlier.
